src/teste.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path');
+
+const Moenda = require(path.resolve(__dirname, './core/moenda.js'));
+
+const config = {
+  h1: {p: 'required', h2: 'required'},
+  h2: [{p: 'required', code: 'optional', h3: 'optional'}, {h3: 'required'}],
+  h3: [
+    {p: 'required', code: 'required'},
+    {p: 'required', table: 'required'},
+  ],
+};
+
+const moenda = new Moenda({
```

Comment:
I used this file to test my changes, I left here because I would like that you can check how the things work together, as soon as we merge, I intend to delete this file.

Reply:
keep it in repository. add as a npm script (test) and use a better filename.